ElevenLabs-DotNet/Voices/CategoryTypes.cs

```diff
@@ -12,22 +12,22 @@ public enum CategoryTypes
         /// <summary>
         /// Premade voice.
         /// </summary>
-        [EnumMember(Value = "premade")]
+        [JsonStringEnumMemberName("premade")]
```

Comment:
Pretty sure there is a way we can do this w/o changing this from `EnumMember` I thought I had a custom converter just for this purpose. I can take a look.